openlibrary/catalog/marc/parse.py

```diff
@@ -357,20 +378,32 @@ def read_publisher(rec):
     return edition
 
 
-def read_author_person(f):
-    f.remove_brackets()
+def name_from_list(name_parts: list[str]) -> str:
+    STRIP_CHARS = r' /,;:[]'
+    name = ' '.join(strip_foc(s).strip(STRIP_CHARS) for s in name_parts)
+    return remove_trailing_dot(name)
+
+
+def read_author_person(f, tag='100') -> dict:
```

Comment:
What is the expected daratype of f?
```suggestion
def read_author_person(f, tag: str = '100') -> dict:
```

Reply:
Done in bf5511b4348e89a143ffc99553cd7e4e2a6b0485